app/src/main/java/fr/free/nrw/commons/contributions/ContributionsFragment.java

```diff
@@ -528,6 +528,15 @@ public void onLocationChangedMedium(LatLng latLng) {
         updateClosestNearbyCardViewInfo();
     }
 
+    @Override
+    public void changeDialogState(boolean enableDialog) {
```

Comment:
This seems to be code used during development. Would it be useful to keep during production too?

Reply:
> This seems to be code used during development. Would it be useful to keep this in production too?

Can you please explain more about this comment? Are some classes development specific only? If so, I didn't know that was the case.

Reply:
Sorry for note being clear. AFAIK, there's no development specific class. What I actually meant to ask was: Is it worth reataining this log prints which just captures whether or not the dialog is enabled? Would it help us somewhere? If not, it can just be removed, right?

Anyways, I suppose it doesn't matter anymore as you're planning to [remove this state](https://github.com/commons-app/apps-android-commons/pull/3438#issuecomment-595335034).

Reply:
> What I actually meant to ask was: Is it worth retaining these log prints which just captures whether or not the dialog is enabled? Would it help us somewhere?

Point noted :nerd_face: 

>Anyways, I suppose it doesn't matter anymore as you're planning to remove this state.

Yup